craft_application/commands/other.py

```diff
@@ -29,6 +30,7 @@ def get_other_command_group() -> CommandGroup:
     """Return the lifecycle related command group."""
     commands: list[type[base.AppCommand]] = [
         InitCommand,
+        LintCommand,
```

Comment:
Let's not make this a default command yet - apps should opt into this command since it only becomes useful once the app has at least one linter.